packages/console-types/src/types/provider.ts

```diff
@@ -77,10 +76,27 @@ const gitProviderCapabilitySchema = {
   },
 } as const
 
+const ciCdToolProviderCapabilitySchema = {
+  type: 'object',
+  additionalProperties: false,
+  required: [
+    'name',
+  ],
+  properties: {
+    name: { const: CI_CD_TOOL },
+    functionalities: providerFunctionalitiesSchema,
+    projectDeployPipelineNameTemplate: { type: 'string' },
+    servicesPipelineNameTemplate: { type: 'string' },
+    apiBaseUrlPathTemplate: { type: 'string' },
+  },
+} as const
+
 export const providerCapabilitySchema = {
-  if: isGitProviderCapability,
```

Comment:
is `isGitProviderCapability` still used anywhere else?

Reply:
yes, in [provider types](https://github.com/mia-platform/console-sdk/blob/f78d9c6cb097111b9dd4073ad628f6c26568e489/packages/console-types/src/types/providerType.ts#L94)

Reply:
we actually could remove the export from there, is not needed anymore

Reply:
done